src/Magnum/Platform/WindowlessEglApplication.h

```diff
@@ -610,6 +646,7 @@ class WindowlessEglApplication {
         #ifndef MAGNUM_TARGET_WEBGL
         /* These are saved from command-line arguments */
         UnsignedInt _commandLineDevice;
+        bool _commandLineDeviceIsCuda = false;
```

Comment:
Similarly here it would be 

```suggestion
        UnsignedInt _commandLineDevice;
        UnsignedInt _commandLineCudaDevice = ~UnsignedInt{};
```

to have the same handling in both